src/AlertCounter.cpp

```diff
@@ -67,7 +67,15 @@ void AlertCounter::inc(time_t when, AlertableEntity *alertable) {
           );
   #endif
 }
+/* *************************************** */
+
+void AlertCounter::inc_no_time_window(){
+  current_hits++;
+}
 
+void AlertCounter::dec(){
+  current_hits--;
+}
```

Comment:
Here you have to check that current_hits is higher than 0.
It can happen in many ways that the first packet of a captured flow does not starts as a normal connection (for instance the packet is forged and starts with a SYN and FIN set or simply the flows in captured half-way during the real connection) and this might bring a counter overflow issue.